Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Expander/ExpanderPage.xaml

```diff
@@ -27,7 +26,7 @@
                     </Grid>
 
                     <controls:Expander.ContentOverlay>
-                        <Grid Background="Red">
+                        <Grid Background="Red" Height="250">
```

Comment:
It seems to work better if I set `MinHeight` instead of `Height`.

Reply:
Good catch, updated